cmd/operator/main.go

```diff
@@ -196,7 +196,8 @@ func main() {
 		setupLog.Error(err, "error removing invalid annotations from Linux nodes")
 	}
 
-	configMapReconciler, err := controllers.NewConfigMapReconciler(mgr, clusterConfig, watchNamespace)
+	proxyEnabled := len(cluster.ClusterWideProxyVars) > 0
```

Comment:
Please add a `proxyEnabled()` function in the cluster package to contain this logic.